app/views/projects/reports.html.erb

```diff
@@ -4,82 +4,143 @@
       <%= link_to @project.name, @project %>
     </li>
   </ul>
-
-  <% since_options.each do |qty| %>
-    <% if params[:since] == qty.to_s %>
-      <%= link_to t("projects.reports.n months ago", count: qty), "#", class: "btn btn-warning btn-sm navbar-btn navbar-btn-reports" %>
-    <% else %>
-      <%= link_to t("projects.reports.n months ago", count: qty), reports_project_path(@project, since: qty), class: "btn btn-warning btn-sm navbar-btn navbar-btn-reports" %>
-    <% end %>
-  <% end %>
-  <% if params[:since].present? %>
-    <%= link_to t('projects.reports.all_time'), reports_project_path(@project), class: "btn btn-warning btn-sm navbar-btn navbar-btn-reports" %>
-  <% end %>
 <% end %>
 
 <% content_for :navbar do %>
   <%= render 'projects/navbar', project: @project %>
 <% end %>
 
 <div class="row">
-  <div class="col-sm-12 col-md-12">
-    <% calculate_and_render_burn_up! %>
-
-    <h3><%= t('projects.reports.burn_up') %></h3>
-    <p>The current backlog has a total of <strong><%= @total_backlog_points %> points</strong>. This is without counting the stories in the Chilly Bean!</p>
-    <p>At the current velocity (<%= @service.velocity %> points) the existing backlog should only be finished around the iteration <%= last_iteration_number %> (<%= last_iteration_start_date %>).</p>
-
-    <% if last_iteration_number == last_iteration_number(true) %>
-      <p>If the velocity keeps up this pace we should be on track to the date above.</p>
-    <% else %>
-      <p>But with a standard deviation of <%= "%3.2f" % Math.sqrt(@service.standard_deviation) %>, it's more realistic to expect the project to finish around <strong>iteration <%= last_iteration_number(true) %></strong> (<%= last_iteration_start_date(true) %>), if the velocity doesn't increase or the backlog is not simplified.</p>
-    <% end %>
-
-    <p><%= area_chart @group_by_day, max: @total_backlog_points, xtitle: t('day'), ytitle: t('accepted_points') %></p>
+  <div class="col-xs-12">
+    <div class="page-header">
+      <h4 class="page-header-title darkgrey-2">
```

Comment:
Can't we add a new class to this header (like`reports-page-header`) and add the color there instead of using a class just for the text color? We should think in element in form of components.